openlibrary/plugins/openlibrary/js/markdown-editor/index.js

```diff
@@ -0,0 +1,14 @@
+// unversioned.
+import '../../../../../vendor/js/wmd/jquery.wmd.js'
```

Comment:
There's also a `legacy-wmd.less`; can that be put here or should that not be loaded dynamically?

Reply:
Looked into this and sadly not a quick fix so have cut a new bug - https://github.com/internetarchive/openlibrary/issues/2370